dumpling-overview.md

```diff
@@ -0,0 +1,257 @@
+| -t or --threads | The number of concurrent backup threads | 4 |
+| -r or --rows | Divide the table into specified rows of data (generally applicable for concurrent operations of splitting a large table into multiple files. |
+| -L or --logfile | Log output address. If it is empty, the log will be output to the console | "" |
+| --loglevel | Log level {debug,info,warn,error,dpanic,panic,fatal} | "info" |
+| --logfmt | Log output format {text,json} | "text" |
+| -d or --no-data | Do not export data (suitable for scenarios where only the schema is exported) |
+| --no-header | Export CSV files of the tables without generating header |
+| -W or --no-views| Do not export the views | true |
+| -m or --no-schemas | Do not export the schema with only the data exported |
+| -s or--statement-size | Control the size of the `INSERT` statements; the unit is bytes |
+| -F or --filesize | The file size of the divided tables. The unit must be specified such as `128B`, `64KiB`, `32MiB`, and `1.5GiB`. |
+| --filetype| Exported file type (csv/sql) | "sql" |
+| -o or --output | Exported file path | "./export-${time}" |
+| -S or --sql | Export data according to the specified SQL statement. This command does not support concurrent export. |
+| --consistency | flush: use FTWRL before the dump <br/> snapshot: specify the dump files' location through TSO <br/> lock: execute `lock tables read` on all tables to be dumped <br/> none: dump without adding locks, which sacrifices consistency <br/> auto: MySQL flush, TiDB snapshot | "auto" |
+| --snapshot | snapshot TSO; valid only when `consistency=snapshot` |
+| --where | Specify the scope of the table backup through the `where` condition |
+| -p or --password | The password of the linked node |
+| -P or --port | The port of the linked node | 4000 |
+| -u or --user | The username of the linked node | "root" |
+| --dump-empty-database | Export the `CREATE DATABASE` statements of the empty databases | true |
+| --tidbMemQuotaQuery | Maximum memory used by a single query when exporting TiDB database | 34359738368(32GB) |
+| --ca | The address of the certificate authority file for TLS connection |
+| --cert | The address of the client certificate file for TLS connection |
+| --key | The address of the client private key file for TLS connection |
+| --csv-delimiter | Delimiter of character type variables in CSV files | '"' |
+| --csv-separator | Separator of each value in CSV files | ',' |
+| --csv-null-value | Representation of null values in CSV files | "\\N" |
+| --escape-backslash | Use backslash (`\`) to escape special characters in the export file | true |
+| --output-filename-template | The data file names in the [golang arguments](https://golang.org/pkg/text/template/#hdr-Arguments) format <br/> Support the `{{.DB}}`, `{{.Table}}`, and `{{.Index}}` arguments <br/> The three arguments represent the database name, table name, and block ID of the data file | '{{.DB}}.{{.Table}}.{{.Index}}' |
```

Comment:
s/golang arguments/golang templates
s/block/chunk

Reply:
I switch `golang templates` to `golang arguments` in https://github.com/pingcap/docs-cn/pull/4096/files#diff-6f3d525218a463bc3b428acd35c6d223R248 because `in the format of the golang arguments` is more appropriate. What do you think? @lichunzhu

Reply:
But actually here we should fill in `go template` language. There is no `golang arguments`.

Reply:
Okay, got. Then I'll update it to `The data file names in the format of golang template`. Is it accurate?

Reply:
`The filename templates represented in the format of golang template` This one